app/models/guide.rb

```diff
@@ -123,6 +127,10 @@ def resettable?
     usage_in_organization.resettable?
   end
 
+  def exercise_assignments_for(user)
```

Comment:
This one seems a bit odd here, but I know it's required by flow and with the current structure there isn't a better place to implement it, so :+1:

Reply:
┐(︶▽︶)┌